picoquic/picosocks.c

```diff
@@ -476,7 +476,8 @@ int picoquic_sendmsg(SOCKET_TYPE fd,
                 cmsg_2->cmsg_level = IPPROTO_IPV6;
                 cmsg_2->cmsg_type = IPV6_DONTFRAG;
                 cmsg_2->cmsg_len = CMSG_LEN(sizeof(int));
-                *((int *)CMSG_DATA(cmsg_2)) = val;
+//                *((int *)CMSG_DATA(cmsg_2)) = val;
```

Comment:
This line can be removed?

Reply:
Yes. It was causing a weird message about pointer mappings in  gcc, so I replaced the mapping by a memcpy.